### DISCUSSION TITLE
Video conferencing recording

### DISCUSSION THREAD

**Discussion by bhantegv** [mynaparrot/plugNmeet-server#423] [Q&A]
Hello.
So I Share Screen then Webcam. When I record the conversation between two or three people with webcam the webcam is not shown in recording. Only audio is recorded. 

Also when having a conversation the webcams cannot be seen either party. Sometimes there is a red circle on the video of the other participant.

I'm not sure what I'm doing wrong. The aim is to have a zoom style discussion with other participants and record interview for upload to social media.

Any help or guidance is appreciated.
Thank you.

**Comment by jibon57** [comment]
Which browser are you using? There have one problem with firefox browser with livekit. If your open screen share first & webcam later then sometime webcam sharing failed without showing any error message. 

Also when having a conversation the webcams cannot be seen either party. Sometimes there is a red circle on the video of the other participant.
> Did you enabled End-to-End encryption feature? 

Only audio is recorded.
> Are you using local or cloud recording?

**Comment by bhantegv** [comment]
Hello.
1. I am using Brave Browser on Android.
2. Yes I enabled end to end encryption.
3. I have tried both local and cloud recording.

I have sent you an email to support, I would like to hire a technician.
Thank you.

**Reply by jibon57** [reply]
I am using Brave Browser on Android.
> Please use Firefox, Google Chrome or Safari browser. You can read this recommendation from readme file.

Yes I enabled end to end encryption.
> Not all the browser support this feature. Check readme file.

**Reply by bhantegv** [reply]
Ok.
Does end to end encryption affect Webcam recording?

**Reply by jibon57** [reply]
That will affect the user who is sharing media e.g webcam, audio, screen share etc. So, the other user may not receive correct stream to display/play. Here recorder is also a user.
